### DISCUSSION TITLE
Need for JSON Schema Draft 7?

### DISCUSSION THREAD

**Discussion by ChakshuGautam** [Sunbird-RC/community#5] [General]
I trying to figure out the internals of Sunbird and wanted to check why the newest Draft of JSON Schema was used rather than the older ones. 

Thanks.

@dileepbapat @vrayulu

**Comment by pramodkvarma** [comment]
@ChakshuGautam do you see an issue?

**Reply by ChakshuGautam** [reply]
@pramodkvarma  Not really no. I wanted to know more about the new features of JSON Schema that Sunbird-RC might be using and the design choices behind the same. I would personally go with a previous spec since a larger number of libraries would support that. But it might be missing a lot of features of validations etc.

**Reply by pramodkvarma** [reply]
OK, got it. 

@dileepbapat @tejash-jl how much of new spec are we depending on? Just curious.
